roles/openshift_node_dns/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install NetworkManager
+  action: "{{ ansible_pkg_mgr }} name=NetworkManager state=present"
+  
+- name: Switch resolver to dnsmasq
+  copy:
+    src: origin-cluster-resolver.conf
```

Comment:
Should we be copying this file over as is, or should we just use the ini_file module to ensure it's there and has the values we expect?